src/CSSBattle/9_Tesseract/WeiPin.html

```diff
@@ -0,0 +1,91 @@
+<div class='rectangle'></div>
+<div class='bsquare'></div>
+<div class='ssquare'></div>
+<div class='circle'></div>
+<style>
+  body {
+    background: #222730;
+    margin:0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .rectangle {
+    position: absolute;
+    width: 400;
+    height: 150px;
+    background:#4CAAB3;
+    margin: 75 0 
+  }
+  .bsquare {
+    position: absolute;
+    width: 250px;
+    height: 250px;
+    background: #222730;
+    transform: rotate(-45deg);
+    margin: 20 72
+  }
+  .ssquare {
+    position: absolute;
+    width: 150px;
+    height: 150px;
+    background: #4CAAB3;
+    transform: rotate(-45deg);
+    margin: -50 30
+  }
+  .circle {
+    position: absolute;
+    width: 50px;
+    height: 50px;
+    background: #393E46;
+    border-radius: 50px;
+  }
+  .fdark3 {
+    position: absolute;
+    width: 20px;
+    height: 140px;
+    background: #060F55;
+    border-radius: 15px;
+    margin: -50 80 
+  }
+  .fdark4 {
+    position: absolute;
+    width: 20px;
+    height: 140px;
+    background: #060F55;
+    border-radius: 15px;
+    margin: -50 120 
+  }
+  .handle {
+    position: absolute;
+    width: 20px;
+    height: 150px;
+    background: #060F55;
+    border-radius: 15px;
+    margin: 60;
+  }
+  .g1 {
+    position: absolute;
+    width: 20px;
+    height: 140px;
+    background: #6592CF;
+    border-radius: 15px;
+    margin: -80 0 0 20;
+  }
+  .g2 {
```

Comment:
same comment! can abstract out same properties into a class like g, then for 1 to 3 make it ID or "g one" "g two" "g three" so less repeating code